comp/safety/incident-record.yaml

```diff
@@ -1,18 +0,0 @@
-summary: Have a means to record an incident
```

Comment:
I don't think we should drop this, it's a "must" task. IIRC this was added by Scarzy.

I see that there's a mention of a similar thing in `comp/safety/comp-risk-assessments`, however the framing of that task is around the risk _assessment_ rather than stuff at the competition itself. Maybe that task needs its wording changing?

Reply:
We've restructured this so that the action of obtaining the book is part of the risk assessment as it ties into evaluating control measures.

The act of ensuring the log is delivered to the competition has been split into it's own task to avoid holding the risk assessment task open.